compiler/src/main/java/com/squareup/anvil/compiler/AnvilCommandLineProcessor.kt

```diff
@@ -61,6 +65,13 @@ class AnvilCommandLineProcessor : CommandLineProcessor {
         "@MergeComponent or @MergeSubcomponent.",
       required = false,
       allowMultipleOccurrences = false
+    ),
+    CliOption(
```

Comment:
I added this option so we could correctly gate the modes, but have not wired it up in the gradle plugin yet. I figure we can wait on that until everything's fully implemented